asset-transfer-basic/application-go/assetTransfer.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2020 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/hyperledger/fabric-sdk-go/pkg/core/config"
+	"github.com/hyperledger/fabric-sdk-go/pkg/gateway"
+)
+
+func main() {
+	fmt.Println("============ application-golang starts ============")
+	wallet, err := gateway.NewFileSystemWallet("wallet")
+	if err != nil {
+		fmt.Printf("Failed to create wallet: %s\n", err)
+		os.Exit(1)
+	}
+
+	if !wallet.Exists("appUser") {
+		err = populateWallet(wallet)
+		if err != nil {
+			fmt.Printf("Failed to populate wallet contents: %s\n", err)
+			os.Exit(1)
+		}
+	}
+
+	ccpPath := filepath.Join(
+		"..",
+		"..",
+		"test-network",
+		"organizations",
+		"peerOrganizations",
+		"org1.example.com",
+		"connection-org1.yaml",
+	)
+
```

Comment:
for this sample to work on one host, isn't this env variable needed?
`export DISCOVERY_AS_LOCALHOST=TRUE`

can we set all the ENV vars needed at beginning of each go application sample?

Reply:
note #232 PR, added instruction to set DISCOVERY_AS_LOCALHOST

Reply:
thanks for pointing it out. this was the cause of fabcar golang application not runnable. 
I added `os.Setenv("DISCOVERY_AS_LOCALHOST", "true")` to the beginning of `assetTransfer.go`, but still want to test it with new deployment script to confirm.